ocaml/xapi/xapi_globs.ml

```diff
@@ -752,6 +752,8 @@ let server_cert_path = ref (Filename.concat "/etc/xensource" "xapi-ssl.pem")
 let server_cert_internal_path =
   ref (Filename.concat "/etc/xensource" "xapi-pool-tls.pem")
 
+let c_rehash = ref "/usr/bin/c_rehash"
```

Comment:
For executables in standard locations, just add the name without full path (`ref "c_rehash"`) and add an entry to `essential_executables` below instead (I see we missed that for the yum/rpm commands).